pkg/release/config_test.go

```diff
@@ -25,6 +26,14 @@ func (s *ConfigTestSuite) TestConfigUniq() {
 	s.Require().NoError(r.Uniq().Validate())
 }
 
+func (s *ConfigTestSuite) TestConfigUniqTags() {
+	r := release.NewConfig()
+
+	r.BuildAfterUnmarshal()
+
+	s.Require().True(slices.Contains(r.TagsF, r.Uniq().String()))
+}
```

Comment:
please test with some name, namespace and context

Reply:
I saw that release.NewConfig() has some filled values.